src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -28,6 +28,34 @@ ActorProfile En_Mag_Profile = {
 
 static s16 sDelayTimer = 0;
 
+#if OOT_VERSION < OOT_GC_US
```

Comment:
there are examples all over of this, but most of them are for matching. its even what they did in the original code to, for example, group together certain items or item actions `(i <= ITEM_POCKET_CUCCO) || (i == ITEM_ODD_MUSHROOM)`

however this shouldnt really have any bearing on our choice here.

I still think this is fine though, it allows for elegantly grouping changes together. Its not like its arbitrary, the enum can be in release order so that they go up sequentially by when different builds released. It lets us group changes among groups of builds without the conditions becoming really long. This will be needed more and more as we get to the bug fixes between the early versions.

Reply:
I didn't intend this as an abuse, the enums are in build order so that they roughly correspond to when changes actually occurred. But here we'd have to list out 7 different versions once N64 versions are added (at least, more if we include 0.9 and gateway) and I was trying to avoid doing that for each of these cases.

There's not really cases like this yet, this is the PR that would set that precedence

Reply:
I think this is good

Reply:
ok clearly outvoted, disregard :)